bot.py

```diff
@@ -38,6 +39,7 @@ def __init__(self):
         self.username = 'Karma Chameleon'
         self.emoji = ':lizard:'
         self.verification_token = os.environ.get('VERIFICATION_TOKEN')
+        self.logger = logging.getLogger('karma_chameleon.bot')
```

Comment:
Why the `.bot` extension? Why not just `.log`?

Reply:
That isn't an extension, it's a name for the logger.  You can see that the master logger has the name "karma_chameleon".  The naming convention, I believe, allows for this module to log to the same logger so that we don't have to redo config or prefs, or worry about file write collisions.